LICENSE

```diff
@@ -186,16 +186,16 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+  Copyright 2019 Maintainers of plus3it/cookiecutter-tfmodule
```

Comment:
As employees, all our rights under copyright go to Plus3. In the U.S., the `(c)` isn't necessary but may be useful in other countries.
```suggestion
  Copyright (c) 2019 Plus3 IT Systems.
```

Reply:
So, the idea was that there may be outside contributors, and signaling that this really is an open source project and that we encourage, understand, and accept open source contributions and what that really entails. Happy to adjust the language with that in mind...

Reply:
If there are outside contributors, nobody cares about them. Okay, that's not true. Fine. I like this:
```suggestion
  Copyright (c) 2019 Plus3 IT Systems and the cookiecutter-tfmodule contributors.
```
We should decide on a company-wide standard and update accordingly.

Reply:
Updated to the most recent recommendation